app/main.py

```diff
@@ -1,3 +1,43 @@
-def shop_trip():
-    # write your code here
-    pass
+import datetime
+from app.shop import Shop
+from app.customer import Customer
+from app.unpack_json_file import list_of_shops, list_of_customers
+
+
+def shop_trip() -> None:
+    for customer_data in list_of_customers:
+        customer = Customer(**customer_data)
+        print(f"{customer.name} has {customer.money} dollars")
+        choice_shop = ""
+        num = 1000000
```

Comment:
No it is wrong set to None and do check as
if num is None or customer.money < num: